docs/features/runs/17_yaml-configs/2_yaml-api.md

```diff
@@ -30,10 +30,20 @@ compute:
     datastore_mount_dir: null     # Where to mount the datastore
     use_spot: false               # If we should use spot instances
     framework: "lightning"        # Which framework to use
+    dependency_file_info:         
+      package_manager: pip        # Either conda or pip
+      path: null                  # Path to the reqs or conda env
 
     # Pass in environment variables
     environment:
       MY_ENVIRONMENT_VARIABLE: "example"
+  
+hyper_params:
+  # your hyper params here
+  params:
+    # your params here
+    --foo: 1
+    -bar: 2
```

Comment:
@oojo12 is the single dash in `-bar` intentional?

Reply:
@essiequoi yes it is.

Reply:
It is meant to test that we correctly parse both the -- and - syntax when used in the config yaml. I took that one from the a test in the source code